### ISSUE TITLE
README contains outdated usage

### ISSUE THREAD

**Issue by longtimeago** [jcabi/jcabi-http#74]
```
    String html = new JdkRequest("https://www.google.com")
      .uri().path("/users").queryParam("id", 333).back()
      .method(Request.GET)
      .header(HttpHeaders.ACCEPT, MediaType.TEXT_HTML)
      .fetch()
      .as(HttpResponse.class)
      .assertStatus(HttpURLConnection.HTTP_OK)
      .body();
  }
```
But there is no ```HttpResponse``` class anymore.
There should be ```RestResponse```

**Comment by yegor256** [comment]
@longtimeago fixed in e159043, thanks!

**Comment by longtimeago** [comment]
@yegor256 you are welcome
